caluma/form/serializers.py

```diff
@@ -612,7 +592,7 @@ def update(self, instance, validated_data):
         for answer_document in models.Document.objects.filter(
             pk__in=answer_documents.values("document")
         ):
-            children = _get_document_tree(answer_document.pk)
+            children = self._get_document_tree(answer_document.pk)
             models.Document.objects.filter(pk__in=children).update(
```

Comment:
Out of context for this PR, but why do we detach the table children from the toplevel family? This doesn't make any sense IMHO, and puts rocks in our path when fetching the document for validation etc

Reply:
#524